src/quak/_widget.py

```diff
@@ -37,7 +39,10 @@ def __init__(self, data, *, table: str = "df"):
             conn = data
         else:
             conn = duckdb.connect(":memory:")
```

Comment:
Do you know if the DuckDB `:memory:` DB will spill to disk? Or for very large input would it just crash the process?

Reply:
> Do you know if the DuckDB :memory: DB will spill to disk? Or for very large input would it just crash the process?

I believe it spills to disk

Reply:
I thought I was talking to someone recently who asserted `:memory:` in particular didn't spill to disk because it doesn't have a path to store a local database. But I'm not sure

Reply:
Hi just stopping by;

in-memory databases will spill to disk, it will create/use the `.tmp` folder in the current working directory, a database file is not required to spill to disk.

You can test this out by setting a relatively low memory limit and creating a temp table that exceeds this limit.
Using `select * from duckdb_temporary_files()` will show you which temporary files were created to back this temporary table